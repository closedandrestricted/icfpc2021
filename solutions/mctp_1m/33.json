{"vertices":[[1,71],[3,0],[3,34],[33,17],[34,52]]}