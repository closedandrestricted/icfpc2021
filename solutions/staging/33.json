{"vertices":[[2,31],[33,17],[33,51],[3,34],[2,69]]}