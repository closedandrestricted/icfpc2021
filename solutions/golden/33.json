{"vertices":[[2,39],[33,53],[33,19],[3,36],[2,1]]}